packages/x-license/src/useLicenseVerifier/useLicenseVerifier.test.tsx

```diff
@@ -63,6 +64,7 @@ describe('useLicenseVerifier', function test() {
         licensingModel: 'perpetual',
         orderNumber: '12345',
         scope: 'pro',
+        planVersion: 'Q3-2024',
```

Comment:
Let's use `'initial'` in the existing tests and use `Q3-2024` in the newly added tests